docs/api/cases/cases-api-set-default-connector.asciidoc

```diff
@@ -0,0 +1,232 @@
+[[cases-api-set-default-connector]]
+== Set default connector for cases API
+++++
+<titleabbrev>Set default connector</titleabbrev>
+++++
+
+Sets the default connector for cases.
+
+=== {api-request-title}
+
+`POST <kibana host>:<port>/api/cases/configure`
+
+`POST <kibana host>:<port>/s/<space_id>/api/cases/configure`
+
+=== {api-prereq-title}
+
+You must have `all` privileges for the *Cases* feature in the *Management*,
+*{observability}*, or *Security* section of the
+<<kibana-feature-privileges,{kib} feature privileges>>, depending on the
+`owner` of the cases.
+
+=== {api-description-title}
+
+Connectors are used to interface with external systems. Before you can set the
+default connector for your cases, you must create it.
+//Refer to <<register-connector>>.
+
+If you do not set a default connector, you can simply provide the connector
```

Comment:
The original text was "You can also set the default connector in the Elastic Security UI for each case individually (see update case...)"

The part that is unclear to me about this is that I can't see what the benefit of defining a "default" connector is.  In my tests, I was *required* to specify a connector in the "create case" API (irrespective of whether I had a default connector)--so there's no benefit there.  And when I update a case, my assumption was that it would keep its existing connector unless otherwise specified... so where does the "default" come into play?

Reply:
I think it means that if you have set a connector on the configuration page then when you create a case this connector will be automatically selected for you.

Reply:
Thanks, I've updated the description as follows:
> If you set a default connector, it is automatically selected when you create
cases in {kib}. If you use the <<cases-api-create,create case API>>, however,
you must still specify all of the connector details.